judgels-frontends/raphael/src/routes/contests/contests/single/scoreboard/ContestScoreboardPage/ContestScoreboardPage.jsx

```diff
@@ -210,17 +232,28 @@ export class ContestScoreboardPage extends React.Component {
     }
 
     const { data: scoreboard, profilesMap } = response[0];
+    const { canViewOtherContestantSolution } = response[0].config;
```

Comment:
Can we destructure `config` first in line 234? To avoid duplication on `response[0]`.